x/group/server/query_server.go

```diff
@@ -42,7 +43,8 @@ func (s serverImpl) getGroupAccountInfo(ctx types.Context, accountAddress sdk.Ac
 }
 
 func (s serverImpl) GroupMembers(ctx types.Context, request *group.QueryGroupMembersRequest) (*group.QueryGroupMembersResponse, error) {
-	it, err := s.getGroupMembers(ctx, request.GroupId, request.Pagination)
+	groupID := group.ID(request.GroupId)
+	it, err := s.getGroupMembers(ctx, groupID, request.Pagination)
```

Comment:
I guess we could also just use `uint64` in the place of `group.ID` as param of `getGroupMembers`